.github/workflows/example-fix.yaml

```diff
@@ -32,6 +33,13 @@ jobs:
         files: ${{ steps.changed-files.outputs.all_changed_files }}
         fixFiles: true
         format: "sarif"
+    # Use HollowMan6/sarif4reviewdog to support SARIF format for reviewdog
+    - name: PR Suggester by SARIF file
+      if: github.event_name == 'pull_request_target'
+      uses: HollowMan6/sarif4reviewdog@v1.0.0
```

Comment:
@craigbox Should we move this action to the Kubescape org?

Reply:
This action is not bonded to kubescape and can act as a general converter from sarif to rdf, but I'm OK if you want to do so.